Makefile

```
@@ -17,6 +17,13 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	targets/openshift/yq.mk \
 )
 
+# Run unit tests in legacy modules as part of test-unit
+test-unit: test-unit-legacy
```

checkmake (0.3.2)

[warning] Target "test-unit" should be declared PHONY.

(phonydeclared)